packages/block-library/src/image/edit.js

```diff
@@ -269,6 +282,18 @@ export function ImageEdit( {
 				value={ align }
 				onChange={ updateAlignment }
 			/>
+			<ToolbarGroup>
```

Comment:
This might make sense in `image/image.js` BlockControls. You can move it then at the end as you mention in PR's description.

Also  we have to take into account if it will be available for multiple selected images (`multiImageSelection`). If yes, that makes sense to be, we have to change `replaceBlocks` to get the selected images `clientIds` and change the `cover/transforms.js` to allow multiple transformations: `isMultiBlock: true` and change the `transform` function accordingly.

If yes

Reply:
Good point. We should disable this feature for multi-selection.